bin/start.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+bundle install
+rm tmp/pids/server.pid 2> /dev/null
```

Comment:
we don't have a tmp/pids directory.  What is this trying to do?

Reply:
Oops... this should be `.internal_test_app/pids/server.pid`...